packages/svelte2tsx/src/svelte2tsx/nodes/ImplicitTopLevelNames.ts

```diff
@@ -50,6 +55,29 @@ export class ImplicitTopLevelNames {
         // in situations where there is a line break of ; guaranteed to be present (else the code is invalid)
     }
 
+    private isNodeStartsWithObjectLiteral(node: ts.Node) {
+        if (ts.isObjectLiteralExpression(node)) {
+            return true;
+        }
+
+        if (ts.isElementAccessExpression(node)) {
+            return this.isNodeStartsWithObjectLiteral(node.expression);
+        }
+
+        if (ts.isBinaryExpression(node)) {
+            return this.isNodeStartsWithObjectLiteral(node.left);
+        }
+
+        if (ts.isConditionalExpression(node)) {
+            return this.isNodeStartsWithObjectLiteral(node.condition);
+        }
+
+        return node
```

Comment:
Should we have this fallback? Or we can just return false here. And fix it later if someone actually brings up the case I can't think of now.

Reply:
The fallback here looks reasonable, so I think we can leave it like this